### PR TITLE
Add MySQL extension to Insider catalog

### PR DESCRIPTION
**Pull request by rishky-msft**
- This adds the [MySQL extension](https://github.com/microsoft/azuredatastudio-mysql) to the Insider build catalog.

- VSIX File for MySQL Extension: [MySQL Extension VSIX 0.2.1](https://github.com/microsoft/azuredatastudio-mysql/releases/download/v0.2.1/azuredatastudio-mysql-0.2.1.vsix)

### CODE REVIEW

File: extensionsGallery-insider.json [hunk 1]
```diff
@@ -4755,6 +4755,67 @@
...
+								}
+							],
+							"properties": [
+								{
+									"key": "Microsoft.VisualStudio.Code.ExtensionDependencies",
+									"value": ""
+								},
+								{
+									"key": "Microsoft.VisualStudio.Code.Engine",
+									"value": ">=1.39.0"
```
Comment (alanrenmsft) [inline extensionsGallery-insider.json:4804]: 1.40.0 is the current version of insider, I believe you have some dependencies on changes made post 1.39.0 release.

Comment (alanrenmsft) [reply extensionsGallery-insider.json:4804]: also, this is for vscode version, you can leave it as * and then update the Microsoft.AzDataEngine to >=1.40.0

File: extensionsGallery-insider.json [hunk 2]
```diff
@@ -4755,6 +4755,67 @@
...
+					"displayName": "MySQL",
+					"shortDescription": "MySQL extension for Azure Data Studio",
+					"publisher": {
+						"displayName": "Microsoft",
+						"publisherId": "Microsoft",
+						"publisherName": "Microsoft"
+					},
+					"versions": [
+						{
+							"version": "1.0.0",
```
[the remark this answers is not in the document]
Comment (rishky-msft) [reply extensionsGallery-insider.json]: Updated to 0.2.0

File: extensionsGallery-insider.json (hunk 1 above)
Comment (rishky-msft) [reply extensionsGallery-insider.json]: Done

**Review by alanrenmsft — APPROVED** [review]

**Review by alanrenmsft — APPROVED** [review]

**Pull request merged**